src/.gitignore

```diff
@@ -0,0 +1,3 @@
+# If we are using symbolic links for dependencies don't check the links in.
```

Comment:
Please mention Arduino here. People who have no idea why these got added shouldn't have to track that info down.

Reply:
I disagree, I considered that and other people may have to do this too.  *Any* embedded environment will need stuff like this.  Besides, you *should* be including stdint.h.